fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -56,8 +56,8 @@
 
     private final AsyncHttpClient webClient;
     private final ObjectWriter writer;
-    private final Meter httpErrorMeter;
-    private final Meter exceptionMeter;
+    protected final Meter httpErrorMeter;
+    protected final Meter exceptionMeter;
```

Comment:
Rather than exposing fields directly, we should use methods instead. In this case, methods to count errors, etc. are probably the right call, and accessors will likely work for the other variables made protected in this pr